token/services/network/common/filter.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package common
+
+import (
+	driver2 "github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
+	token3 "github.com/hyperledger-labs/fabric-token-sdk/token"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/auditdb"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/ttxdb"
+	"github.com/pkg/errors"
+)
+
+type TransactionFilterProvider[F driver2.TransactionFilter] interface {
+	New(tmsID token3.TMSID) (F, error)
+}
+
+type AcceptTxInDBFilterProvider struct {
```

Comment:
Maybe a comment to describe more or less what this filter is used for

Reply:
👍